core/microgame/metadata/mg_definition.gd

```diff
@@ -0,0 +1,12 @@
+extends Resource
+class_name MicrogameDefinition
+
+enum InputType {
+    MASH, # TODO - i don't love this for arrows+button but it seems to be the (WarioWare) Gold standard
+}
```

Comment:
I usually have the last type of an enum be "SIZE" and have found that it's quite a useful bookend in an enum. 
- You could query how many input types you have (InputType.SIZE) 
- It feels like you're restricting the enum on one end - you can add things before SIZE, but not after it. Which is helpful when enums show in editor dropdowns because reordering an enum messes up what a dropdown is set to.